### PR TITLE
Add example of $edit:arg-completer for git

### PR DESCRIPTION
**Pull request by miller-time**

### CODE REVIEW

File: src/ref/edit.md [hunk 1]
```diff
@@ -259,6 +259,44 @@ edit:arg-completer[apt] = [@args]{
 }
 ```
 
+Here is another example of configuring a completer. This time for the `git`
+command. It supports completing some common subcommands and then branch
+names after that:
+
+```elvish
+fn all-git-branches {
+    e:git branch -a --format="%(refname:strip=2)" | eawk [0 1 @rest]{ put $1 }
```
Comment (miller-time) [inline src/ref/edit.md:275]: Just discovered that the `--format="%(refname:strip=2)"` argument doesn't work on older versions of git (it errors on 2.10.2). Works fine after upgrading to 2.15.0, though. I tried searching through the git release notes and couldn't figure out why.

Comment (xiaq) [reply src/ref/edit.md:275]: It's fine for education purpose; but please leave a comment here explaining that this assumes a recent version of git.

The `e:` in `e:git` is not needed.

**Comment by xiaq** [comment]
Sorry - I missed this PR for a long time!

**Comment by xiaq** [comment]
This has been integrated in https://github.com/elves/elvish/commit/3fe8533c8553d1262c6a3b64dd9858a02dfff979

**Pull request closed without merging**